src/components/Search/Search.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from "react";
+import axios from "axios"
+import './Search.scss'
+
+const Search = ({title, setWeatherInfo}) => {
+    const [city, setCity] = useState("")
+   
+    const API_KEY = process.env.REACT_APP_API_KEY
+    console.log('API Key:', API_KEY);
+    const API_URL = `http://api.openweathermap.org/data/2.5/forecast?q=${city}&cnt=8&units=metric&appid=${API_KEY}`
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        
+        //API Request using Axios
+        getWeatherInfo()
+        console.log(city);
+    }
+
+    // This function sends an API Request via Axios
+    const getWeatherInfo = async () => {
+        // axios request
+    // try {
+
+    // }
+    // catch(err){
+
+    // }
```

Comment:
Avoid committing comments like this, unless they are particularly useful to the other developers working on the project with you.